src/adapters/discord.js

```diff
@@ -0,0 +1,87 @@
+const Discord = require('discord.js');
+const debug = require('debug')('bot:adapter:discord');
+
+const isString = require('lodash/isString');
+const isObject = require('lodash/isObject');
+const isArray = require('lodash/isArray');
+const isEmpty = require('lodash/isEmpty');
+
+const { discord: discordCfg } = require('../config');
+
+const discordAdapter = () => {};
```

Comment:
\>_>
Looks like it might be a `class` :)

Reply:
in js functions and objects a first-class citizens... also for only `__INIT__` cases i prefer
```
module.exports = {
  __INIT__(context) {
  // mutation here
  return context;
 }
}
```
but need rework app.js in future for support AND i open for discussions, maybe some class sugar or another syntax will be better